### PR TITLE
PLA-6590-CLI-Refresh-to-get-results

### PR DESCRIPTION
**Pull request by zechenjiang**
Removed unsupported command, migrated from govendor to dep, added two new command, coreo result rule and coreo result object.
Currently we output all the information returned by webapp. Maybe change this in the future.

### CODE REVIEW

**Comment by coolguru** [comment]
@zechenjiang can you please fix issues reported by travis-ci and also run gofmt on your changes.

File: client/result.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package client
+
+import (
+	"context"
```
Comment (coolguru) [inline client/result.go]: format please

File: cmd/result_rule.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"github.com/CloudCoreo/cli/pkg/coreo"
```
Comment (coolguru) [inline cmd/result_rule.go]: format please

File: client/result.go (hunk 1 above)
Comment (zechenjiang) [reply client/result.go]: Np, I'll do that and add comments to meet gometalinter requirements right now.

**Pull request merged**